docs/testing/cypress.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Testing with Cypress
+description: Use Cypress to write end-to-end tests with Clerk.
+---
+
+# Testing with Cypress
+
+<Callout type="info">
+  Our `@clerk/testing` package supports only end-to-end testing with Cypress. Unit tests are not supported.
+</Callout>
+
+Cypress is a well-established JavaScript testing framework. This guide aims to help you set up your environment for creating authenticated tests with Clerk. This guide will assume you're somewhat familiar with Clerk and Cypress.
+
+Before diving in, you might want to start by visiting the [Cypress starter](https://github.com/clerk/example-cypress-nextjs) for an example repo that tests a Clerk-powered application using [Testing Tokens](/docs/testing/overview#testing-tokens).
+
+<Steps>
+### Install `@clerk/testing`
+
+Clerk's testing package provides integration helpers for popular testing frameworks.
+
+```bash filename="terminal"
+npm install @clerk/testing --save-dev
+```
+
+### Set your API keys
+
+Set your publishable and secret keys in your test runner, as the `CLERK_PUBLISHABLE_KEY` and `CLERK_SECRET_KEY` environment variables respectively.
+
+<Callout type="warning">
+  Ensure you provide the secret key in a secure manner, to avoid leaking it to third parties. For example, if you are using GitHub Actions, refer to [*Using secrets in GitHub Actions*](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).
+</Callout>
```

Comment:
Let's use our new syntax as to avoid tech debt!

```suggestion
> [!WARNING]
> Ensure you provide the secret key in a secure manner, to avoid leaking it to third parties. For example, if you are using GitHub Actions, refer to [*Using secrets in GitHub Actions*](https://docs.github.com/en/actions/security-guides/using-secrets-in-github-actions).
```